packages/fxa-content-server/app/scripts/views/subscriptions_redirect.js

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import FormView from './form';
+import Template from 'templates/subscriptions_redirect.mustache';
+
+class SubscriptionsRedirectView extends FormView {
+  mustAuth = true;
+  template = Template;
+
+  initialize (options) {
+    this._currentPage = options.currentPage;
+    this._config = {};
+    if (options && options.config && options.config.subscriptions) {
+      this._config = options.config.subscriptions;
+    }
+  }
+
+  afterRender () {
+    const {
+      managementClientId,
+      managementScopes,
+      managementTokenTTL,
+      managementUrl,
+    } = this._config;
+    return this.getSignedInAccount()
+      .createOAuthToken(managementScopes, {
+        client_id: managementClientId, //eslint-disable-line camelcase
+        ttl: managementTokenTTL,
+      })
+      .then((accessToken) => {
+        const searchQuery = window.location.search;
```

Comment:
Can you use a reference to `this.window.location.search`, it makes unit testing easier.